### ISSUE TITLE
Common `step`, `min`, `max` parameters for Point input

### ISSUE THREAD

**Issue by Lutymane** [cocopon/tweakpane#417]
Hello,

It'll be very handy to be able to specify the mentioned parameters in options when both dimensions have same constrains instead of duplicating them for `x` and `y`.

So current way is:
```js
const PARAMS = {
  offset: {x: 20, y: 30},
};

const pane = new Pane();
pane.addInput(PARAMS, 'offset', {
  x: {step: 1, min: 0, max: 4},
  y: {step: 1, min: 0, max: 4},
});
```

Desired:
```js
const PARAMS = {
  offset: {x: 20, y: 30},
};

const pane = new Pane();
pane.addInput(PARAMS, 'offset', {
  step: 20, min: 0, max: 4
});
```

So the check would be something like `params.x.step ?? params.step` meaning root level params would act as fallback, while specifying them per property would override defaults

**Comment by alijaya** [comment]
For workaround I guess you could do it like this:
```
const opts = {step: 20, min: 0, max: 4};
pane.addInput(PARAMS, 'offset', {
  x: opts, // in case using all default value
  y: {...opts, step: 1} // in case want to override it
});
```

**Comment by cocopon** [comment]
Implemented in #514 and will be released in the next major version.

**Comment by cocopon** [comment]
Released in v4.0.0.
